src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -245,7 +260,7 @@ private SchemaKeyspace()
               + "PRIMARY KEY ((keyspace_name), aggregate_name, argument_types))");
 
     private static final List<TableMetadata> ALL_TABLE_METADATA =
-        ImmutableList.of(Keyspaces, Tables, Columns, Triggers, DroppedColumns, Views, Types, Functions, Aggregates, Indexes);
+        ImmutableList.of(Keyspaces, Tables, Columns, ColumnMasks, Triggers, DroppedColumns, Views, Types, Functions, Aggregates, Indexes);
```

Comment:
formatting

Reply:
Do you want me to change it to:
```java
    private static final List<TableMetadata> ALL_TABLE_METADATA = ImmutableList.of(Keyspaces,
                                                                                   Tables,
                                                                                   Columns,
                                                                                   ColumnMasks,
                                                                                   Triggers,
                                                                                   DroppedColumns,
                                                                                   Views,
                                                                                   Types,
                                                                                   Functions,
                                                                                   Aggregates,
                                                                                   Indexes);
```
or use double tab:
```java
    private static final List<TableMetadata> ALL_TABLE_METADATA =
            ImmutableList.of(Keyspaces, Tables, Columns, ColumnMasks, Triggers, DroppedColumns, Views, Types, Functions, Aggregates, Indexes);
```
?

Reply:
I would opt for the first option to follow the current style?